client/src/components/admin/users-list/AdminComment.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="card">
+    <div class="card-body card-text">
+      <pre class="mb-0">{{ comment.content }}</pre>
+    </div>
+    <div class="card-footer">
+      <span class="text-muted">
+        {{ formatTimestamp(comment.timestamp) }} &ndash; {{ authorName }}
+      </span>
+      <button v-if="isOwnComment" @click="deleteComment">
+        <font-awesome-icon icon="trash" class="text-muted"/>
+      </button>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from 'vue';
+import { Comment } from '@/store/types/Comment';
+import Student from '@/models/Student';
+import { LOAD_USER } from '@/store/actions';
+
+const DAY_MILLIS = 24 * 60 * 60 * 1000;
+
+export default defineComponent({
+  props: {
+    comment: {
+      type: Object as PropType<Comment>,
+      required: true,
+    },
+  },
+  emits: ['deleteComment'],
+  data() {
+    return {
+      author: undefined as Student | undefined,
+    };
+  },
+  computed: {
+    authorName(): string {
+      return `${this.author?.firstName} ${this.author?.lastName}`;
+    },
+    dateFormatter(): Intl.DateTimeFormat {
+      return new Intl.DateTimeFormat([this.$i18n.locale, 'en'], { day: '2-digit', month: '2-digit', year: 'numeric' });
```

Comment:
Wir haben in helpers/dateHelper auch schon eine normalizeDate Funktion, vielleicht kannst du sie benutzen oder die Date/File Sachen auch in das File verschieben?

Reply:
Würde ich ungern machen, weil das im server-package liegt 😅 Aber die Funktion macht auch was völlig anderes. Falls Du das trotzdem gerne an einem Ort hättest, müssten wir dafür ein neues Package anlegen, wo dann common code liegt. Aber ich glaube, das lohnt sich hier nicht wirklich